server/internal/config/config.go

```diff
@@ -216,3 +272,343 @@ func getNetwork() (common.Network, error) {
 		return common.Network{}, fmt.Errorf("unknown network %s", viper.GetString(Network))
 	}
 }
+
+func DetermineLocktimeType(locktime int64) common.RelativeLocktime {
```

Comment:
Should not be exported :
```suggestion
func determineLocktimeType(locktime int64) common.RelativeLocktime {
```